src/app/catalog/product/store/product.effects.ts

```diff
@@ -38,9 +37,7 @@ export class ProductEffects {
       ofType(loadProductDetails),
       switchMap(({ productId }) =>
         this.service.getProduct(productId).pipe(
-          switchMap((product) =>
-            of(loadProductDetailsSuccess({ product }), navigate({ url: `/product/${product.id}` }))
-          ),
+          switchMap((product) => of(loadProductDetailsSuccess({ product }))),
```

Comment:
Can this be a simple `map()`?
```suggestion
          map((product) => loadProductDetailsSuccess({ product })),
```

Reply:
changed